Python/Product/Analysis/Infrastructure/StringExtensions.cs

```diff
@@ -16,15 +16,68 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
 using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.PythonTools.Analysis.Infrastructure {
     static class StringExtensions {
```

Comment:
Why not make this class `public`? It can be useful everywhere in PTVS.

Reply:
It's InternalsVisibleTo. `public` API is public/stable for end users, especially in the `Analysis` project which we know will have multiple users.

Reply:
Ok, got it